CreateSnapshot/src/test/resources/log4j2.properties

```
@@ -0,0 +1,18 @@
+status = WARN
+
+property.ownedPackagesLogLevel=${sys:migrationLogLevel:-INFO}
+
+appender.console.type = Console
+appender.console.name = Console
+appender.console.target = SYSTEM_OUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} %p %c{1.} [%t] %m%n
+
+rootLogger.level = info
+rootLogger.appenderRef.console.ref = Console
+
+# Allow customization of owned package logs
+logger.rfs.name = com.rfs
+logger.rfs.level = ${ownedPackagesLogLevel}
```

Seems like it would be useful to fallback to debug, so the build.gradle only needs to specify the level for `test` when running with trace enabled.  What do you think about making this to all the log4j files?
```suggestion
logger.rfs.level = ${ownedPackagesLogLevel:-DEBUG}
```